certified-connectors/FactSet/settings.json

```diff
@@ -0,0 +1,9 @@
+{

```

Comment:
@krithsai We do not require icon.png and settings.json files in the open-source repo, please remove them from the PR.